Composer/packages/server/package-lock.json

```diff
@@ -0,0 +1,1780 @@
+{
```

Comment:
we should also remove this

Reply:
Man, I accidentally run one 'npm' command instead of 'yarn' and these files sneak in there. Ill remove now.